pkg/resourcemanager/util/util.go

```diff
@@ -59,4 +59,6 @@ const (
 	CheckTable
 	// ImportInto is for import into component.
 	ImportInto
+	// COPROCESSOR is for handling coprocessor tasks.
+	COPROCESSOR
```

Comment:
How about to use `Coprocessor`? All other consts are in camel case, except for UNKONWN, which is only used in testing.